### DISCUSSION TITLE
再请问大佬使用GitHub actions进行自动构建二进制的问题

### DISCUSSION THREAD

**Discussion by xunzh2000** [AlistGo/alist#670] [Q&A]
大佬好，按[https://github.com/Xhofe/alist/issues/164](https://github.com/Xhofe/alist/issues/url)想重构二进制，fork后[https://github.com/xunzh2000/alist/](https://github.com/Xhofe/alist/issues/url)改了[https://github.com/xunzh2000/alist/blob/v2/build.sh](https://github.com/Xhofe/alist/issues/url)但是actions重构后，[https://github.com/xunzh2000/alist/suites/5523210484/artifacts/177284207](https://github.com/Xhofe/alist/issues/url)alist Linux二进制好像前端不可用，前端不显示，空白，标签页只有一个图标。请问下anhui老乡，哪里的问题，谢谢，有点小白啊。

**Comment by xhofe** [comment]
自己编译只能使用：https://alist-doc.nn.ci/docs/setting/config#%E6%9C%AC%E5%9C%B0

**Reply by xunzh2000** [reply]
谢谢大佬

**Comment by xunzh2000** [comment]
搞不了。算了吧
